packages/hint/docs/user-guide/concepts/connectors.md

```diff
@@ -103,12 +103,21 @@ The set of settings specific for them are:
 * `tabUrl (string)`: The URL to visit before the final target in case
   `useTabUrl` is `true`. `https://empty.webhint.io/` is the
   default value.
+* `flags? (Array<string>)`: Allows you to pass in additional Chrome
+  command line API flags. Useful if you would like to start your
+  session in headless mode or with GPU disabled. Here's the full list
+  of [available command line flags][cli flags].
+* `waitForContentLoaded (number)`: Time the browser has to wait for the
```

Comment:
Use the same text as before and maybe put a note that this is only for browsers using the debugging protocol?

Reply:
This is already in a section for remote debugging protocol.